estimagic/logging/read_database.py

```diff
@@ -0,0 +1,164 @@
+"""Functions to read from a database."""
```

Comment:
There is a discrepancy here between the level of generality in the module docstring ("a database") and the two first functions (read rows of one very specific table in a very specific database).

Reply:
Changed it. The more general functions are not public (because they still require some knowledge of sqlalchemy), so in fact the module only contains specific functions.